kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue

```diff
@@ -47,10 +47,31 @@
           :key="`forget-${idx}`"
           :text="$tr('forgetAddressButtonLabel')"
           appearance="basic-link"
-          @click="removeAddress(a.id)"
+          @click="removeSavedAddress(a.id)"
         />
       </div>
     </template>
+
+    <hr v-if="discoveredAddresses.length > 0">
```

Comment:
in subsequent work, let's make sure this is styled as a one-pixel solid line using the theme color `$themeTokens.fineLine`